src/manifest.json

```
@@ -1,7 +1,7 @@
 {
   "name": "__MSG_extName__",
   "description": "__MSG_extDesc__",
-  "version": "19.7.14.946",
+  "version": "20.3.16.1532",
```

```suggestion
  "version": "0.0.0",
```

This is set automatically by the publisher

---

My bad, I thought we had automated the deployment of this extension, but it's still manual 🤔